test/python/common.py

```diff
@@ -140,14 +140,14 @@ def assertDictAlmostEqual(self, dict1, dict2, delta=None, msg=None,
                     standard_msg += '(%s: %s != %s), ' % (safe_repr(key),
                                                           safe_repr(val1),
                                                           safe_repr(val2))
-            if success is True:
+            if success:
                 return
             standard_msg = standard_msg[:-2] + ' within %s places' % places
 
         else:
             if delta is None:
-                delta = 1e-8  # default delta value
-            success = True
+                delta = 0.04 * sum(dict1.values())  # default delta value
```

Comment:
I don't think you should change the default delta value here. The more lenient value you put in for fixing tests (`0.04 * shots`) should be added in the tests themselves so it's obvious when reading the test what threshold you are using.